### ISSUE TITLE
[Bash Completion] Generated bash completion code doesn't work for paths with spaces

### ISSUE THREAD

**Issue by rednoah** [remkop/picocli#1546]
Here's my picocli Java code:
```java
	@Option(names = "--output", description = "Output path")
	public File output;
```

Here's the generated bash completion code:
```sh
    --output)
      compopt -o filenames
      COMPREPLY=( $( compgen -f -- "${curr_word}" ) ) # files
      return $?
      ;;
```

The solution is to add `local IFS=$'\n'` before each `compgen -f -- "${curr_word}"`:
https://github.com/filebot/plugins/commit/2e475164add69bb7591d5f70d839f9a66e171ba1

I see that in the demo already in the repository here, but the latest `4.6.2` doesn't seem to generate it like that for me, and I couldn't see any recent change notes for that:
https://github.com/remkop/picocli/blob/d99afd2638acc14e6c54f99d2a81dd91d51d9ef5/src/test/resources/picocompletion-demo_completion.bash#L258

**Comment by remkop** [comment]
Hi @rednoah , this has been fixed in #1458 and #1473 but has not been released yet.

**Comment by remkop** [comment]
Closing as duplicate. This will be fixed in the next release.
